build/source/driver/multi_driver.f90

```diff
@@ -369,69 +525,88 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
 do istep=1,numtim
 
  ! set print flag
- globalPrintFlag=.false.
+ globalPrintFlag=.true.
 
  ! read a line of forcing data (if not already opened, open file, and get to the correct place)
- ! NOTE: only read data once: if same data used for multiple HRUs, data is copied across
  do iHRU=1,nHRU  ! loop through HRUs
-  ! assign pointers to HRUs
-  time_data => time_hru(iHRU)
-  forc_data => forc_hru(iHRU)
-  ! read forcing data
-  call read_force(istep,iHRU,err,message); call handle_err(err,message)
+  if(forcFileInfo(iHRU)%ixFirstHRU > 0)then
+   forcStruct%hru(iHRU) = forcStruct%hru(forcFileInfo(iHRU)%ixFirstHRU)  ! copy forcing data from another HRU
+  else
+   call read_force(istep,iHRU,timeStruct%var,forcStruct%hru(iHRU)%var,err,message); call handle_err(err,message)
+  endif
  end do  ! (end looping through HRUs)
 
  ! print progress
- if(globalPrintFlag)then
-  if(time_data%var(iLookTIME%ih) == 1) write(*,'(i4,1x,5(i2,1x))') time_data%var
- endif
+ !if(globalPrintFlag)then
+  !if(timeStruct%var(iLookTIME%ih) == 1) write(*,'(i4,1x,5(i2,1x))') timeStruct%var
+  write(*,'(i4,1x,5(i2,1x))') timeStruct%var
+ !endif
+
+ ! compute the exposed LAI and SAI and whether veg is buried by snow
+ if(istep==1)then  ! (call phenology here because we need the time information)
```

Comment:
Good point. The routine `vegPhenlgy` also computes some time information, which is used before `coupled_em` is called for the first time. The comment needs to be more descriptive.